docs/site/FAQ.md

```diff
@@ -64,6 +117,28 @@ projects).
 For more details, see the lengthy discussion in
 [#6](https://github.com/strongloop/loopback-next/issues/6).
 
+## About the documentation
+
+### Where are the tutorials?
+
+See [Examples](Examples.md) and [Tutorials](Tutorials.md).
+
+### What features are planned?
+
+Our roadmap is captured in
+[here](https://github.com/strongloop/loopback-next/labels/roadmap). You can also
+check out our
+[monthly milestone plans](https://github.com/strongloop/loopback-next/labels/Monthly%20Milestone).
+
+For LoopBack 3 users, here is the list of LB3 feature parity we wish to be
+implemented in the future:
+https://github.com/strongloop/loopback-next/issues/1920.
+
+Do you have some other features in mind? Add your feature requests at
+[loopback-next/issues/new](https://github.com/strongloop/loopback-next/issues/new).
+
+## Technical usage
+
 ### Does JavaScript still work?
 
 LoopBack 4 itself is written in [TypeScript](https://www.typescriptlang.org)
```

Comment:
> LoopBack 4 itself is written in [TypeScript](https://www.typescriptlang.org)
(that compiles to JavaScript), but it supports applications written in both
TypeScript and JavaScript. The documentation assumes you have a basic
understanding of the JavaScript language; and when it says "JavaScript" it is
understood to mean ECMAScript version 6 (ES6).

I am afraid LB4 does not support JavaScript, we require our users to write their code in TypeScript. See #560 and the discussion in #1978 for more information on the challenges involved.